paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-a', '--from', '--after', dest='time_from',
```

Comment:
I kinda like `--to` and `--from`

`--since` and `--to` don't go together nearly as nicely.